### ISSUE TITLE
Hidden + Freeze Frame causes hit circles to approach at the same time

### ISSUE THREAD

**Issue by kingmigdor** [ppy/osu#21104]
### Type

Game behaviour

### Bug description

It's likely impossible to see those circles approach since all circles disappear at the same time. Hit circles should rather disappear when you need to click them than disappear when the first circle needs to be clicked.

### Screenshots or videos

https://user-images.githubusercontent.com/67754395/199788511-54cd6b67-363e-43ad-aba5-dbdca2baf8b2.mp4

### Version

2022.1101.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9931468/database.log)
[input.log](https://github.com/ppy/osu/files/9931469/input.log)
[network.log](https://github.com/ppy/osu/files/9931473/network.log)
[performance.log](https://github.com/ppy/osu/files/9931474/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9931475/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9931478/updater.log)

**Comment by Doxxz** [comment]
Doesn't Hidden's "Only fade approach circles" setting mitigate this already?

**Comment by kingmigdor** [comment]
> this was intentional on my part atleast, and the general reception has been good towards the hidden combination, with people saying it provides a fun challenge, i myself have enjoyed learning maps with HDFR

You cannot really see when to click the circle, that's the thing

**Comment by mk56-spn** [comment]
> > this was intentional on my part atleast, and the general reception has been good towards the hidden combination, with people saying it provides a fun challenge, i myself have enjoyed learning maps with HDFR
> 
> You cannot really see when to click the circle, that's the thing

I don't think this is an issue. They can be hit correctly through a mix of ryrhmic understanding of the map, memory and pattern recognition. Not everything has to be  sight-readable

**Comment by bdach** [comment]
Judging from the reactions on the opening post I don't think this is an unanimous bug that 100% needs addressing. Converting to discussion until consensus is reached one way or another.
